public/docs/dart/latest/guide/attribute-directives.jade

```diff
@@ -1,12 +1,376 @@
 include ../_util-fns
 
 :marked
-  We're working on the Dart version of this chapter.
-  In the meantime, please see these resources:
+  An **Attribute** directive changes the appearance or behavior of a DOM element.
 
-  * [Attribute Directives](/docs/ts/latest/guide/attribute-directives.html):
-    The TypeScript version of this chapter
+:marked
+  In this chapter we will
+  * write an attribute directive to change the background color
+  * apply the attribute directive to an element in a template
+  * respond to user-initiated events
+  * pass values into the directive using data binding
+
+  [Live Example](/resources/live-examples/attribute-directives/ts/plnkr.html)
```

Comment:
Please replace this with a link to the Dart source, as in the Template Syntax page.